students/arkadiusz_kasprzyk/lesson_06_dicts/guess_the_number.py

```diff
@@ -0,0 +1,32 @@
+import random
+
+
+def guess(N):
+    print("Give your guess (number between 1 " +
+          "and {} or one of '0', '?', 'h', 'help' for help):".format(N))
+    guesset = input("").split(" ")
+    if guesset[0].lower() in {'0', '?', 'h', 'help'}:
+        geusset = set()
+    else:
+        guesset = set(guesset) - set("")
+        geusset = set([int(num) for num in guesset])
```

Comment:
Also again you can use split without parameters to skip multiple spaces and newlines handling and `map(int, guesset)` is a natural fit here.